.gitignore

```diff
@@ -66,6 +66,9 @@ node_modules/
 codekit-config.json
 .pycharm_helpers/
 
+.project
+.pydevproject
```

Comment:
That is a suggestion from the git team/authors: http://git-scm.com/docs/gitignore brought up by @bdero to ignore those files globally.